### PR TITLE
App ready

### PR DESCRIPTION
**Pull request by GiVi713**

### CODE REVIEW

**Comment by GiVi713** [comment]
FIRST PART OF HW 12 . Second part named "add logger and log commands"

File: HW_12/HW_12/Program.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (VadzimPapko) [inline HW_12/HW_12/Program.cs:37]: uh

File: HW_12/HW_12/Randomizer.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HW_12
+{
+    class Randomizer
```
Comment (VadzimPapko) [inline HW_12/HW_12/Randomizer.cs:7]: interesting idea

**Review by VadzimPapko — APPROVED** [review]
code is not clean in this task. your score is 8

**Pull request merged**